lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -695,6 +695,8 @@ class BarTouchTooltipData with EquatableMixin {
     double? tooltipRoundedRadius,
     EdgeInsets? tooltipPadding,
     double? tooltipMargin,
+    FLHorizontalAlignment? tooltipAlignment,
```

Comment:
It's better to rename `tooltipAlignment` to `tooltipHorizontalAlignment` to make it more descriptive.
Thanks

Reply:
Don't forget to apply it on all 3 TooltipData classes

Reply:
Done